public/js/main.js

```diff
@@ -8,3 +21,8 @@ window.addEventListener("load", () => {
     // Call setCallbacks to register Interactive Canvas
     window.scene.action.setCallbacks();
 });
+
+// Get the header height of the device and set the top body padding accordingly
+window.interactiveCanvas.getHeaderHeightPx().then((headerHeight) => {
```

Comment:
This is good, I should use this in my code as well. I will import when we merge :)